test/proposals/DeployMarketCreationProposal.s.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.19;
+
+import {console} from "@forge-std/console.sol";
+import {Script} from "@forge-std/Script.sol";
+
+import {Addresses} from "@test/proposals/Addresses.sol";
+import {mip0x as mip} from "@test/proposals/mips/examples/mip-market-listing/mip-market-listing.sol";
```

Comment:
It appears that this is hardcoded to the example MIP. How can we make this dynamic like the other 3 env vars? Or is it impossible?

Reply:
not possible to make it dynamic. leverage the example mip because it is the code we're never supposed to change and we can just sent env vars